server/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -121,6 +121,9 @@ private ContainerTag<?> renderBlockWithSubmitForm(Params params) {
         .withAction(formAction)
         .withMethod(HttpVerbs.POST)
         .with(makeCsrfTokenInputTag(params.request()))
+        .with(
+            requiredFieldsExplanationContent()
```

Comment:
Sorry I missed this initially, but we'll need to use the messages API instead of using a hardcoded English string. This part: https://github.com/civiform/civiform/blob/2a3664c6477d0dca4996104a243b6bba8210206e/server/app/views/BaseHtmlView.java#L150-L153

should be updated. Here's an example PR of how to do that:
https://github.com/civiform/civiform/pull/3564/files

And can you actually split that out into a different PR? Based on the slack thread, we should submit it for translation first, then block this PR until it's been translated.

Our very official process for translations is to add the strings and then ping Bion :D